frontend/src/components/board/header/ToggleReactionsButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { ListItemIcon, ListItemText, MenuItem } from "@material-ui/core";
+import { EmojiEmotions } from "@material-ui/icons";
+import React, { useContext } from "react";
+import { TOGGLE_REACTIONS } from "../../../constants/event.constants";
+import { BoardContext } from "../../../context/BoardContext";
+import { UserContext } from "../../../context/UserContext";
+import { ROLE_MODERATOR } from "../../../utils/user.utils";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Quickly check if `eslint` is still complaining when you remove this line. You have a usage in the export statement, therefore I see no problem, but I could be wrong.

Reply:
Worked! :)